src/lib.rs

```diff
@@ -0,0 +1,108 @@
+pub mod config {
+    use serde::Deserialize;
+
+    use std::error::Error;
+    use std::fs::File;
+    use std::io::BufReader;
+    use std::path::Path;
+
+    #[derive(Deserialize, Debug)]
+    pub struct Config {
+        pub db_address: String,
+    }
+
+    pub fn read_json_config(path: impl AsRef<Path>) -> Result<Config, Box<dyn Error>> {
+        let file = File::open(path)?;
+        let reader = BufReader::new(file);
+
+        // read json from file
+        let json_data = serde_json::from_reader(reader)?;
+
+        Ok(json_data)
+    }
+}
+
+pub mod db {
+    use chrono::{serde::ts_seconds, serde::ts_seconds_option, DateTime, Utc};
```

Comment:
入れ子にuseするならこうじゃないすかね

```suggestion
    use chrono::{serde::{ts_seconds, ts_seconds_option}, DateTime, Utc};
```

Reply:
acc18947d647274631f6860e89d9fa24db41b25d
done